packages/patternfly-4/react-core/src/components/AboutModal/AboutModalBox.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import styles from '@patternfly/patternfly-next/components/AboutModalBox/styles.css';
+
+const propTypes = {
+  /** content rendered inside the AboutModelBox. */
+  children: PropTypes.node.isRequired,
+  /** additional classes added to the AboutModalBox */
+  className: PropTypes.string,
+  /** id to use for Modal Box aria-labelledby attribute */
+  ariaLablledbyId: PropTypes.string.isRequired,
```

Comment:
nit: typo

Reply:
Why not call it 'aria-labelledby' here and elsewhere? And 'aria-describedby'. Then to make it easier to work with, when they get passed into AboutModalBox could rename there ... className, 'aria-labelledby': ariaLabelledBy, ...props